Dockerfile

```diff
@@ -0,0 +1,138 @@
+    && chmod 777 /opt && chmod a+s /opt \
+    && mkdir -p /neurodocker \
+    && if [ ! -f "$ND_ENTRYPOINT" ]; then \
+         echo '#!/usr/bin/env bash' >> "$ND_ENTRYPOINT" \
+    &&   echo 'set -e' >> "$ND_ENTRYPOINT" \
+    &&   echo 'export USER="${USER:=`whoami`}"' >> "$ND_ENTRYPOINT" \
+    &&   echo 'if [ -n "$1" ]; then "$@"; else /usr/bin/env bash; fi' >> "$ND_ENTRYPOINT"; \
+    fi \
+    && chmod -R 777 /neurodocker && chmod a+s /neurodocker
+
+ENTRYPOINT ["/neurodocker/startup.sh"]
+
+ENV FREESURFER_HOME="/opt/freesurfer-6.0.0" \
+    PATH="/opt/freesurfer-6.0.0/bin:$PATH"
+RUN apt-get update -qq \
+    && apt-get install -y -q --no-install-recommends \
+           bc \
+           libgomp1 \
+           libxmu6 \
+           libxt6 \
+           perl \
+           tcsh \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && echo "Downloading FreeSurfer ..." \
+    && mkdir -p /opt/freesurfer-6.0.0 \
+    && curl -fsSL --retry 5 ftp://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/6.0.0/freesurfer-Linux-centos6_x86_64-stable-pub-v6.0.0.tar.gz \
+    | tar -xz -C /opt/freesurfer-6.0.0 --strip-components 1 \
+         --exclude='freesurfer/average/mult-comp-cor' \
+         --exclude='freesurfer/lib/cuda' \
+         --exclude='freesurfer/lib/qt' \
+         --exclude='freesurfer/subjects/V1_average' \
+         --exclude='freesurfer/subjects/bert' \
+         --exclude='freesurfer/subjects/cvs_avg35' \
+         --exclude='freesurfer/subjects/cvs_avg35_inMNI152' \
+         --exclude='freesurfer/subjects/fsaverage3' \
+         --exclude='freesurfer/subjects/fsaverage4' \
+         --exclude='freesurfer/subjects/fsaverage5' \
+         --exclude='freesurfer/subjects/fsaverage6' \
+         --exclude='freesurfer/subjects/fsaverage_sym' \
+         --exclude='freesurfer/trctrain' \
+    && sed -i '$isource "/opt/freesurfer-6.0.0/SetUpFreeSurfer.sh"' "$ND_ENTRYPOINT"
+
+ENV CONDA_DIR="/opt/miniconda-latest" \
+    PATH="/opt/miniconda-latest/bin:$PATH"
+RUN export PATH="/opt/miniconda-latest/bin:$PATH" \
+    && echo "Downloading Miniconda installer ..." \
+    && conda_installer="/tmp/miniconda.sh" \
+    && curl -fsSL --retry 5 -o "$conda_installer" https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && bash "$conda_installer" -b -p /opt/miniconda-latest \
+    && rm -f "$conda_installer" \
+    && conda update -yq -nbase conda \
+    && conda config --system --prepend channels conda-forge \
+    && conda config --system --set auto_update_conda false \
+    && conda config --system --set show_channel_urls true \
+    && sync && conda clean --all && sync \
+    && conda create -y -q --name mb-mm \
+    && conda install -y -q --name mb-mm \
+           'python=3.6' \
+           'numpy' \
+           'pandas' \
+           'sklearn-lmer' \
+           'scipy' \
+           'rpy2=2.9.4' \
+           'r-lme4' \
+           'r-lmertest' \
+           'r-lsmeans' \
```

Comment:
lets go with ``emmeans`` instead of ``lsmeans``. I think their going to archive ``lsmeans`` in the near future.

Reply:
kk, thx for the pointer. I will update this part.

Reply:
Done [here](https://github.com/C0C0AN/Mindboggle-MixedModels/pull/2/commits/e63eba433083f2566dae2b717b4c0a1ee201ef64).